wasm-fuzzer/fuzzing-client-afl/entrypoint_afl.sh

```diff
@@ -27,11 +27,17 @@ if [ $? != 0 ]; then
     exit 1
 fi
 
+if [[ $LOCAL_AFL != "True" ]]; then
+    AFL='afl-fuzz'
+fi
+
 # AFL Docs:
 # afl-fuzz starts by performing an array of deterministic fuzzing steps, 
 # which can take several days, but tend to produce neat test cases. 
 # If you want quick & dirty results right away - akin to zzuf and other 
 # traditional fuzzers - add the -d option to the command line.
 
-echo "afl-fuzz -i $DOCKER_AFL_INPUT -o $DOCKER_AFL_OUTPUT $RANK -d -- ${DOCKER_INTERFACE_SRC}/interface.out @@ $REQUIRED_BYTES"
-exec afl-fuzz -i $DOCKER_AFL_INPUT -o $DOCKER_AFL_OUTPUT $RANK -d -- "${DOCKER_INTERFACE_SRC}/interface.out" @@ $REQUIRED_BYTES
+
+echo "$AFL -i $DOCKER_AFL_INPUT -o $DOCKER_AFL_OUTPUT $RANK -d -- ${DOCKER_INTERFACE_SRC}/interface.out @@ $REQUIRED_BYTES"
+$AFL -i "$DOCKER_AFL_INPUT" -o $DOCKER_AFL_OUTPUT $RANK -t 12000 -d -- "${DOCKER_INTERFACE_SRC}/interface.out" @@ $REQUIRED_BYTES
```

Comment:
This timeout should not be in master imo. According to AFL it really screws the performance. The timeout is a multiple of the time taken of the first test cases anyways, so we shouldn't worry about it.